release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3400,6 +3400,39 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.14.50
+[id="ocp-4-14-50_{context}"]
+=== RHSA-2025:3569 - {product-title} 4.14.50 bug fix and security update
+
+Issued: 09 April 2025
+
+{product-title} release 4.14.50, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:3569[RHSA-2025:3569] advisory.
```

Comment:
This [link](https://access.redhat.com/errata/RHSA-2025:3569) 404s here and in the 3568 advisory. I assume the package update advisory is forthcoming?

Reply:
Yeah, https://access.redhat.com/errata/RHSA-2025:3569 now works!